src/api.c

```diff
@@ -537,3 +551,64 @@ API int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd)
 
 	return 0;
 }
+
+/* NOTE - function header comment in include/seccomp.h */
+int seccomp_alloc_notifications(struct seccomp_notif **req,
+				struct seccomp_notif_resp **resp)
+{
+	static struct seccomp_notif_sizes sizes = {};
+
+	if (sizes.seccomp_notif == 0) {
+		int ret = syscall(__NR_seccomp, SECCOMP_GET_NOTIF_SIZES, 0, &sizes);
+		if (ret < 0)
+			return errno;
+	} else {
+		return -1;
+	}
+
+	*req = malloc(sizes.seccomp_notif);
+	if (!*req)
+		return -ENOMEM;
+	memset(*req, 0, sizes.seccomp_notif);
+
+	*resp = malloc(sizes.seccomp_notif_resp);
```

Comment:
Another one.  I'm going to stop pointing these out and just assume you'll change them all :)